src/include/txn_inline.h

```diff
@@ -963,11 +963,8 @@ __wt_upd_alloc(WT_SESSION_IMPL *session, const WT_ITEM *value, u_int modify_type
         upd->size = WT_STORE_SIZE(value->size);
         memcpy(upd->data, value->data, value->size);
     }
-    /*
-     * This field is const but we need to set it once at allocation time, to do so temporarily cast
-     * as a non-const.
-     */
-    *(uint8_t *)&upd->type = (uint8_t)modify_type;
+
```

Comment:
Previously we did not have this whitespace, if you think its better with it we can keep it, otherwise I'd go back to not having it. Not a big deal, will approve PR.

Reply:
My mistake: removed.